packages/animation/src/types/animation.ts

```diff
@@ -126,4 +126,5 @@ export type AnimationInputWithPreset = AnimationInput & {
 export type StoryAnimation = AnimationInputWithPreset & {
   id: string;
   targets: ElementId[];
+  delete?: boolean;
```

Comment:
I don't think this goes here in general? It's only used for that updater thingy and not a property of animations when in use in the story.